docs/dg/dev/integrate-and-configure/integrate-svg-sprite-asset.md

```diff
@@ -0,0 +1,220 @@
+            if (existsSync(path)) {
+                twigContent = await fs.readFile(path, 'utf8');
+                usedPath = path;
+                console.info(`Using icon sprite from: ${path}`);
+                break;
+            }
+        }
+
+        if (!twigContent) {
+            throw new Error('None of the provided icon sprite paths exist');
+        }
+
+        const spacelessRegex = /{% apply spaceless %}([\s\S]*?)(?:{% endapply %}|$)/;
+        const match = twigContent.match(spacelessRegex);
+
+        if (!match || !match[1]) {
+            throw new Error(`Could not find content within spaceless block in the Twig file: ${usedPath}`);
+        }
+
+        const svgContent = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="display: none;">
+${match[1]}
+</svg>`;
+
+        const targetDir = path.dirname(targetPath);
+        await fs.mkdir(targetDir, { recursive: true });
+
+        await fs.writeFile(targetPath, svgContent, 'utf8');
+
+        console.info('Icon sprites successfully extracted to', targetPath);
+    } catch (error) {
+        console.error('Error extracting icon sprites:', error.message);
+    }
+};
+
+module.exports = extractIconSprites;
+```
+
+---
+
+5. Update Frontend Settings
+
+**File:**  
+`frontend/settings.js`
+
+**Add the following inside your `globalSettings` paths:**
+```js
+    iconSprite: {
+        sources: [
+            './src/Pyz/Yves/ShopUi/Theme/default/components/atoms/icon-sprite/icon-sprite.twig',
+            './vendor/spryker-shop/shop-ui/src/SprykerShop/Yves/ShopUi/Theme/default/components/atoms/icon-sprite/icon-sprite.twig',
+        ],
+        target: './frontend/assets/global/default/icons/sprite.svg',
+    },
+```
+**And inside `getAppSettingsByTheme`:**
+```js
+    iconSprite: globalSettings.paths.iconSprite,
+```
+
+---
+
+6. Update the Frontend Build Config
+
+**File:**  
+`frontend/configs/development.js`
+
+**Add or update:**
+```js
+let isExtractIconSpritesEnabled = false;
+let extractIconSprites = null;
+
+try {
+    extractIconSprites = require('../libs/icon-sprite-extractor');
+    isExtractIconSpritesEnabled = true;
+} catch (e) {
+    console.info('Icon sprite extraction is disabled.');
+}
+
+const getConfiguration = async (appSettings) => {
+    // ...existing code...
+
+    if (isExtractIconSpritesEnabled) {
+        try {
+            await extractIconSprites({
+                sourcePath: appSettings.paths.iconSprite.sources.map((src) => join(process.cwd(), src)),
+                targetPath: join(process.cwd(), appSettings.paths.iconSprite.target),
+            });
+        } catch (error) {
+            console.error('Error extracting icon sprites:', error);
+        }
+    }
+
+    // ...rest of your function...
+};
+```
+
+---
+
+7. Update `.gitignore`
+
+Add this line to avoid committing the generated sprite:
+
+```
+/frontend/assets/global/default/icons/sprite.svg
+```
+
+---
```

Comment:
instead of all this complicated integration guide, and a lot of code with console command
can we please recommend copy SVG code, put it to SVG file under the git - and have the same result?

and remove svg from twig of course